<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>231</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelVertical">
      <property name="text">
       <string>Vertical Value Here</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelHorizontal">
      <property name="text">
       <string>Horizontal Value Here</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="btn_VInfo">
          <property name="text">
           <string>V_Info</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_VRange">
          <property name="text">
           <string>V_Range</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="btn_HInfo">
          <property name="text">
           <string>H_Info</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_HRange">
          <property name="text">
           <string>H_Range</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="horizontalSlider_Test">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_Test">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>22</width>
     <height>131</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
